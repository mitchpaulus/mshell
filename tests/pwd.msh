# This expects the PWD from the outer shell to be passed in as first arg
# Should end with 'tests' for this test directory.
args :0: pwd-from-shell!
pwd pwd-from-mshell!
[
    (@pwd-from-shell @pwd-from-mshell =)
    ("PWD is correct" wl)
    ("PWD is not correct" wl)
] if

'..' cd
pwd new-pwd!
@pwd-from-shell "/tests" "" findReplace expected-pwd!
[
    (@new-pwd @expected-pwd =)
    ("New PWD is correct" wl)
    ("New PWD is not correct" wl)
] if
