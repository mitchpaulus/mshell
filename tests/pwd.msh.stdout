PWD is correct
New PWD is correct
